packages/hub/src/lib/cache-management.ts

```diff
@@ -0,0 +1,262 @@
+import { homedir } from "node:os";
+import { join, basename } from "node:path";
+import { stat, readdir, readFile, realpath, lstat } from "node:fs/promises";
+import type { Stats } from "node:fs";
+import type { RepoType } from "../types/public";
+
+function getDefaultHome(): string {
+	return join(homedir(), ".cache");
+}
+
+function getDefaultCachePath(): string {
+	return join(process.env["HF_HOME"] ?? join(process.env["XDG_CACHE_HOME"] ?? getDefaultHome(), "huggingface"), "hub");
+}
+
+function getHuggingFaceHubCache(): string {
+	return process.env["HUGGINGFACE_HUB_CACHE"] ?? getDefaultCachePath();
+}
+
+function getHFHubCache(): string {
+	return process.env["HF_HUB_CACHE"] ?? getHuggingFaceHubCache();
+}
+
+const FILES_TO_IGNORE: string[] = [".DS_Store"];
+
+export interface CachedFileInfo {
+	filename: string;
+	filePath: string;
+	blobPath: string;
+	sizeOnDisk: number;
+
+	blobLastAccessedAt: Date;
+	blobLastModifiedAt: Date;
```

Comment:
I think `filename` is uncessary since we already have `filePath`.

```suggestion
	path: string;
	/**
	 * Underlying file - which `path` is symlinked to 
	 */
	blob: {
		size: number;
		path: string;
		lastModifiedAt: Date;
		lastAccessedAt: Date;
	}
```

and I think that's it! The doc comment is needed in my opinion (unless we can find a better name for `blob`)

Reply:
Applied ✅